js/src/forum/fragments/AutocompleteDropdown.js

```diff
@@ -24,8 +23,10 @@ export default class AutocompleteDropdown extends Component {
   }
 
   hide() {
-    this.$().hide();
-    this.active = false;
+    if (this.element) {
+      this.$().hide();
+      this.active = false;
+    }
```

Comment:
What scenario does this prevent?

Reply:
At times when testing, hide() was called when the element DOM did not exist, so it errored.

Reply:
jQuery calls shouldn't error if the element doesn't exist, no? Calling `.hide()` on an empty set should work just fine.

Reply:
I'll test more